mmseg/datasets/custom.py

```diff
@@ -338,33 +337,44 @@ def evaluate(self, results, metric='mIoU', logger=None, **kwargs):
         else:
             num_classes = len(self.CLASSES)
 
-        all_acc, acc, iou = mean_iou(
-            results, gt_seg_maps, num_classes, ignore_index=self.ignore_index)
+        all_acc, acc, eval_metric = eval_metrics(
+            results,
+            gt_seg_maps,
+            num_classes,
+            ignore_index=self.ignore_index,
+            metric=metric)
         summary_str = ''
         summary_str += 'per class results:\n'
 
         line_format = '{:<15} {:>10} {:>10}\n'
-        summary_str += line_format.format('Class', 'IoU', 'Acc')
+        if metric == 'mIoU':
+            summary_str += line_format.format('Class', 'IoU', 'Acc')
+        else:
+            summary_str += line_format.format('Class', 'Dice', 'Acc')
```

Comment:
Support more than one metric.

Reply:
We may consider using `terminaltables.AsciiTable`. 
Please refer to [here](https://github.com/open-mmlab/mmdetection/blob/d3cf38d91c454b1a6881e8c36c1e4a66dc5521b8/mmdet/datasets/coco.py#L523) for example.

Reply:
Also [here](https://github.com/open-mmlab/mmdetection/blob/d3cf38d91c454b1a6881e8c36c1e4a66dc5521b8/mmdet/core/evaluation/recall.py#L138)